### DISCUSSION TITLE
How to use the feature of "color enhancement for 3D printing" in Meshlab?

### DISCUSSION THREAD

**Discussion by cjccjj** [cnr-isti-vclab/meshlab#1548] [Q&A]
I noticed from the Meshlab website's front-page banner images of the 'color enhancement for 3D printing' feature. By examining the demo images, I believe (and hope) it works similarly to a 2D Sharpen filter in Photoshop, but applied to 3D meshes. This would be perfect for enhancing surface details, like patterns or material textures from a scan, for 3D printing. However, I can't locate this feature or figure out a workflow in Meshlab how to use it, nor find any information about it except for the demo images. Does anyone have any clues?

( P.S.  stackoverflow no longer work for most MeshLab questions, because only codeing related questions are allowed in there. )

![image](https://github.com/user-attachments/assets/b5b7a2cc-7e70-49a6-9fd3-df4a2f1c7bbd)
![image](https://github.com/user-attachments/assets/b764fa23-2f5a-49d2-b44f-12965baf3ebf)

**Comment by cignoni** [comment]
it is the "Unsharp Mask Color" filter, in this case applied to just a simple ambient occlusion per vertex color.
<img width="1159" alt="Screenshot 2025-05-06 at 11 40 13" src="https://github.com/user-attachments/assets/0fb39f6e-af2f-48ba-a96b-da4f59dfee2a" />
